### PR TITLE
Add Etch a Sketch

### PR DESCRIPTION
**Pull request by jfmartinz**
## Related Issue

issue: #500 

## Description

Added a etch a sketch project. tech stack is HTML CSS JAVASCRIPT

## Screenshots

image

image

### CODE REVIEW

File: .env.example
Comment (priyankarpal) [inline .env.example:1]: please don't delete this file

Comment (jfmartinz) [reply .env.example:1]: I just rename it with .env, i never delete it

Comment (priyankarpal) [reply .env.example:1]: > I just rename it with .env, I never delete it

ok ok so

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by jfmartinz** [comment]
> Looks Good 🎉

Hi @priyankarpal  Can you add what level is this pr?

**Pull request merged**